filemgr/src/test/java/org/apache/oodt/cas/filemgr/system/AbstractFileManagerServerTest.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.oodt.cas.filemgr.system;
+
+import junit.framework.TestCase;
+import org.apache.oodt.cas.filemgr.ingest.StdIngester;
+import org.apache.oodt.cas.filemgr.metadata.CoreMetKeys;
+import org.apache.oodt.cas.metadata.Metadata;
+import org.apache.oodt.cas.metadata.SerializableMetadata;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.net.URL;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+public abstract class AbstractFileManagerServerTest extends TestCase {
+
+    private static final Logger LOG = Logger.getLogger(AbstractFileManagerServerTest.class.getName());
```

Comment:
I think we use Slf4j over Log4j... maybe I am wrong.

Reply:
Thanks for the comments @lewismc! Well, the usage of logging is confusing since at some places it has used `java.util.Logging` and in some places it has used `slf4j API` with `log4j-slf4j implementation`. I personally prefer `slf4j`. But in these classes I kept what was being used.

Reply:
Yes it is confusing. OK, please ignore my comment here then and consider the remaining comments, thank you.

Reply:
Sure. Thank you again. I will fix the others. I will try to make the logging consistent in the future since I believe that logging helps a lot when debugging. ;-)